### PR TITLE
doc: add IntRepOfBipartition

### PR DESCRIPTION
**Pull request by mtorpey**
This needed documenting, as per #298.

### CODE REVIEW

File: doc/bipart.xml [hunk 1]
```diff
@@ -177,6 +177,34 @@ gap> CodomainOfBipartition(x);
   </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="IntRepOfBipartition">
+  <ManSection>
+    <Attr Name = "IntRepOfBipartition" Arg = "x"/>
+    <Returns>A list of positive integers.</Returns>
+    <Description>
+      If <A>x</A> is a bipartition with degree <C>n</C>, then
+      <C>IntRepOfBipartition</C> returns the <E>integer representation</E> of
```
Comment (james-d-mitchell) [inline doc/bipart.xml]: `Int` stands for `Internal` not `Integer`

File: doc/bipart.xml [hunk 2]
```diff
@@ -177,6 +177,34 @@ gap> CodomainOfBipartition(x);
...
+      <A>x</A>: a list of length <C>2 * n</C> containing positive integers which
+      correspond to the blocks of <A>x</A>. <P/>
+
+      If <C>i</C> is in <C>[1 .. n]</C> then <C>list[i]</C> refers to the point
```
Comment (james-d-mitchell) [inline doc/bipart.xml]: comma before "then"

File: doc/bipart.xml [hunk 3]
```diff
@@ -177,6 +177,34 @@ gap> CodomainOfBipartition(x);
...
+      <C>i</C>; if <C>i</C> is in <C>[n + 1 .. 2 * n]</C> then <C>list[i]</C>
+      refers to the point <C>n - i</C> (a negative point).  Two points lie in
+      the same block of the bipartition if and only if their entries in the list
+      are equal. <P/>
+
+      See also <Ref Oper="BipartitionByIntRep"/>. <P/>
```
Comment (james-d-mitchell) [inline doc/bipart.xml]: No need for <P/> at the end.

File: doc/bipart.xml [hunk 4]
```diff
@@ -733,7 +761,10 @@ true]]></Example>
       <C>3</C> in the second position.<P/>
 
       In its first form <C>BipartitionByIntRep</C> verifies that the argument
-      <A>list</A> is the internal representation of a bipartition.
+      <A>list</A> is the internal representation of a bipartition. <P/>
+
+      See also <Ref Oper="IntRepOfBipartition"/>. <P/>
```
Comment (james-d-mitchell) [inline doc/bipart.xml]: NNP

**Comment by mtorpey** [comment]
Changed as suggested.

**Pull request merged**